data/mods/Item_Category_Overhaul/categories.json

```
@@ -0,0 +1,265 @@
+    "name": { "str": "OTHER" },
+    "zone": "LOOT_OTHER",
+    "sort_rank": -1
+  },
+  {
+    "id": "fuel",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "FUEL" },
+    "zone": "LOOT_FUEL",
+    "sort_rank": 1
+  },
+  {
+    "id": "seeds",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "SEEDS" },
+    "zone": "LOOT_SEEDS",
+    "sort_rank": 2
+  },
+  {
+    "id": "chems",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "CHEMICAL STUFF" },
+    "zone": "LOOT_CHEMICAL",
+    "sort_rank": 5
+  },
+  {
+    "id": "battery",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "BATTERIES" },
+    "zone": "LOOT_BATTERY",
+    "sort_rank": 6
+  },
+  {
+    "id": "spare_parts",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "SPARE PARTS" },
+    "zone": "LOOT_SPARE_PARTS",
+    "sort_rank": 8
+  },
+  {
+    "id": "scrap_metal",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "METAL SCRAP" },
+    "zone": "LOOT_SCRAP_METAL",
+    "sort_rank": 9
+  },
+  {
+    "id": "scrap_electronics",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "ELECTRONIC SCRAP" },
+    "zone": "LOOT_SCRAP_ELECTRONICS",
+    "sort_rank": 10
+  },
+  {
+    "id": "scrap_fabric",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "FABRICS" },
+    "zone": "LOOT_SCRAP_FABRIC",
+    "sort_rank": 11
+  },
+  {
+    "id": "scrap_wood",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "WOOD SCRAP" },
+    "zone": "LOOT_SCRAP_WOOD",
+    "sort_rank": 12
+  },
+  {
+    "id": "scrap_plastic",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "PLASTIC SCRAP" },
+    "zone": "LOOT_SCRAP_PLASTIC",
+    "sort_rank": 13
+  },
+  {
+    "id": "scrap_ceramics",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "CERAMIC SCRAP" },
+    "zone": "LOOT_SCRAP_CERAMICS",
+    "sort_rank": 14
+  },
+  {
+    "id": "scrap_glass",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "GLASS SCRAP" },
+    "zone": "LOOT_SCRAP_GLASS",
+    "sort_rank": 15
+  },
+  {
+    "id": "valuables",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "VALUABLES" },
+    "zone": "LOOT_VALUABLES",
+    "sort_rank": 16
+  },
+  {
+    "id": "rocks",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "ROCKS" },
+    "zone": "LOOT_ROCKS",
+    "sort_rank": 17
+  },
+  {
+    "id": "soil",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "SOIL" },
+    "zone": "LOOT_SOIL",
+    "sort_rank": 18
+  },
+  {
+    "id": "container",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "CONTAINERS" },
+    "zone": "LOOT_CONTAINERS",
+    "//": "Please don't touch the sort rank, I did this so these are always on bottom.",
+    "sort_rank": 48
+  },
+  {
+    "id": "artifacts",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "ARTIFACTS" },
+    "zone": "LOOT_ARTIFACTS",
+    "//": "Please don't touch the sort rank, I did this so these are always on bottom.",
+    "sort_rank": 49
+  },
+  {
+    "id": "maps",
+    "type": "ITEM_CATEGORY",
+    "name": { "str": "INFORMATION" },
```

```suggestion
    "name": { "str": "INFORMATION" },
```
Given the weirdness with hardcoded city names on any item in this category gaining city names, may want to wait and mess with this after, and let the other items being put in this category stay in "other" for now.